modules/end-user-guide/pages/adding-a-vscode-extension.adoc

```diff
@@ -1,24 +1,34 @@
-:navtitle: Adding a Visual Studio Code Extension
+:navtitle: Adding a Visual Studio Code extension to a workspace
```

Comment:
:navtitle: Adding a Visual Studio Code plug-in to a workspace

Reply:
FYI VS Code has extensions, not plug-ins https://code.visualstudio.com/docs/editor/extension-marketplace